media_ui/src/main/java/edu/artic/media/ui/AudioDetailsFragment.kt

```diff
@@ -143,6 +152,60 @@ class AudioDetailsFragment : BaseViewModelFragment<AudioDetailsViewModel>() {
                 .subscribe {
                     credit.setContentText(it)
                 }.disposedBy(disposeBag)
+
+        viewModel.relatedTours
+                .observeOn(AndroidSchedulers.mainThread())
+                .subscribe { tours ->
+                    if (tours.isEmpty()) {
+                        relatedToursView.visibility = View.GONE
+                        dividerBelowRelatedTours.visibility = View.GONE
+                    } else {
+                        relatedToursView.removeAllViews()
+                        relatedToursView.visibility = View.VISIBLE
+                        dividerBelowRelatedTours.visibility = View.VISIBLE
+                        addRelatedToursToView(tours)
+                    }
+                }
+                .disposedBy(disposeBag)
+    }
+
+    @UiThread
+    fun addRelatedToursToView(tours: List<ArticTour>) {
+        val textView = TextView(requireContext())
+        textView.text = resources.getString(R.string.relatedTours)
+        TextViewCompat.setTextAppearance(textView, R.style.SectionTitleWhite)
```

Comment:
I think this (the title/heading) should be defined in the XML. Would that be ok?

Reply:
Moved it to layout file.